stories/Who_was_protected_by_the_first_COVID-19_lockdown.ipynb

```diff
@@ -0,0 +1,2412 @@
unnzZnXo0KHudrzzzjvqrbfe2uV/JyFE9zRnQFsHDx5UR44cqaqqqj7xxBPqn//8Z/djLX++7LLL1Ndee8392GWXXabu3r27w7/1Rx55RH3hhRdUVVXV0tJSdcKECerixYvPasPixYvV55577qzfV1VVqfn5+erOnTvdbR03bpx64sQJddWqVeq0adPU06dPq6qqqgsWLFBff/11ddmyZepPfvIT9eOPP1ZnzJihFhUVqaqqqn/729/UH/3oR6rValXNZrN67bXXqk888YSqqp7zdPny5eqDDz7obuOECRPUf/7zn6qqqur111+v7tq1S128eLF6+eWXq3V1daqqqup9992nvvLKK97+OYQQPeDQoUPqBRdcoFZXV6u7du1S8/Ly3Nd3gwcPVisrK1VVbZ19HV2z3XbbbepTTz3V7nvJNZwIFJkjK1ppHm730Ucf8dVXX/Huu+/y6KOPcvLkSTZs2EBZWRl33HEHs2fP5he/+AWKonDixIlWx9i8eTMXXXQR55xzDgD5+fmkpKTw3XffATBq1Ch0uqbBAF999RW7du3i+uuvZ/bs2fzjH//g6NGjPre3oaGBHTt2cOuttwIQHx/PddddxzfffMOOHTs477zzGDp0KADTpk3jz3/+MwA2m40f/ehHDB06lPz8fAA2bdrEhAkT3D3Q//M//+N+n2+++YabbroJvV6PRqPh9ttvZ926dfTp04esrCwKCgpYt24dP/nJT9iyZQuqqrJ69Wr3EMWcnBx3O4YNG0ZNTU0n/lWEEMGgKAoGg8Gn544dO7bd33v6W1+7dq07UzIyMpg+fXqn2rZ792769u3LyJEjATjvvPO48MIL2bp1K5s2bWL69OkkJiYC8NRTT3H//fcDTfPEnnjiCW666Says7OBpqy7+uqriYqKIiYmhpkzZwId5+lll13Gli1bcDgcrF+/nvvvv58NGzZQWlpKZWUlI0aMAGDcuHHuHhPJOiFC13vvvcfkyZNJSkoiLy+P3Nxc3n///Q5f4+2azVMueiLXcKK7pJAVHbrwwgsZMGAABQUFuFwu8vPz3YXuRx99xNKlS8+aO6Gq6lnHUVUVh8MBQExMjPv3LpeLe+65x328ZcuW8d577/ncPpfLddb7uVwuHA4HWq0WRVFatWH//v3un//4xz+yZ88e9/xfRVFaHUur1bY6Ztv3sNvtAFxxxRV88803bNiwgenTp5OTk8Onn36KwWCgb9++AK0ujtu+jxAiNBQUFLiHo7X9O23+e2/WMsda8vS3rtPpWh1Po+nc6bdtBsH3udo262prazGZTEDTheE777zDq6++6v5dW81Z11GeJiYmMmzYML766ivq6uqYPXs227ZtY9WqVUydOtX9/pJ1QoS+hoYGVqxYwfbt25kyZQpTpkyhvLycJUuWnJV1LXm7ZvOUix0dT67hRHdIISs6dPToUY4dO8bQoUO56KKL2LBhA4WFhUBTD8OsWbOwWq2tXtP8vJMnTwJNd8mKi4vdPQktTZw4kX/961/U19cD8Morr/D444/73L64uDhGjhzpXlGvrq6OFStWcPHFFzNy5EgKCws5dOgQAKtXr+axxx4DICoqijFjxrBgwQKeeeYZysvLufjii9mwYQMlJSUALF++3P0+l1xyCf/85z+x2+24XC6WLFnChAkTgKYQXLlyJU6nk4yMDCZMmMDvf//7Ti8YI4ToOUePHuX111/nrrvuAiA5Odk9iqSqqopt27Z16/iTJk3iX//6FwDV1dWsWrWq1UWaNyNHjuTo0aPs3r0bgEOHDvHtt98ybtw4Lr74Yr788kt3jr766qv87W9/A6B///7k5+dz++2388QTT+ByubjkkktYsWIFVqsVq9XKp59+CnScpwBTp07lxRdfJD8/n7i4OAYMGMDbb78tWSdEmFm5ciXJycmsW7eONWvWsGbNGlatWkVDQwP/+c9/Wj1Xq9W6OyK6e83WllzDie6SxZ5EKxaLhdmzZ7t/drlczJ8/nwEDBgAwf/58Hn30UVRVRafT8cYbb5x1B27QoEE888wz/OxnP8PpdGIwGHjzzTfdi4m0dOONN1JaWsoPf/hDFEUhOzv7rEVKmhUWFp615Po333zDokWLmD9/Ph9++CE2m829lYaiKCxatIgnnnjCvejUSy+91Or148ePZ8aMGcydO5e3336bxx57jB//+MfExsaSl5fnft7999/PwoULueaaa3A4HOTl5fGrX/3K/XkB9/CWiRMn8vrrr0sIChHCWmadRqMhOjqaRx99lMmTJwNw++2384tf/IIrr7yS3Nxcxo0b1633e+qpp5g3bx4zZ84kKSmJnJwcj8OYly5d2uoibMiQIfzzn//klVde4fnnn8disaAoCr/97W8ZMGAAAwYM4PDhw9x8881AUyY9//zzrVabnzNnDmvWrOHPf/4zd999NydOnODqq68mKSmJfv36uZ/nKU+hqZB9/vnn+cUvfgE0Zd2SJUu48MILu/XdCCGC67333uPOO+9s1WuZkJDA7bffzt///vdWz7300kt5/vnngaatxny9ZmtLruFEICiq9I8LIYQQAbVkyRKGDRvG6NGjsdls3HLLLTz44INMmjSpp5smhBBChCXpkRVCCCECrLmXtHlu1vTp06WIFUIIIbohKD2yCxcupLq6+qzhBytWrGDRokWkpqYCMHnyZB555JFAN0cIIQJCsk4I0RtI1gkhQkHAe2Q3bdrE8uXL3fOOWiooKODJJ5/k6quvDnQzhBAioCTrhBC9gWSdECJUBHTV4tOnT/PSSy8xZ86cdh8vKChgxYoVzJo1i1/84heyL5MQIixJ1gkhegPJOiFEKAloj+zTTz/NI488QnFxcbuPp6en85Of/IS8vDxefPFF5s+fzx/+8Aevx7VYLHz33Xekp6e3WnFNCNG7OZ1OysvLueCCCzyuCBsIknVCiGCSrBNC9Abesi5ghewHH3xAdnY2+fn5fPjhh+0+549//KP7/++55x6mTp3q
```

Comment:
> We believe that the fact that people from working class backgrounds are less likely to be able to work from home made them more prone to contracting the virus...

Could you add a suggestion of what data analysis would need to be done to move this from an observational correlation into an understanding of causation? It feels like a big jump for the conclusions section as it is phrased at the moment. (You also made the same point earlier, but it's clearly identified as a hypothesis and isn't part of your conclusion section so that gets ✅ from me)